appuio/redis/templates/recreate.yaml

```diff
@@ -0,0 +1,76 @@
+{{- if .Values.forceUpdate }}
+---
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: sts-deleter
+  annotations:
+    # This is what defines this resource as a hook. Without this line, the
+    # job is considered part of the release.
```

Comment:
This should be common knowledge in Helm charts. I would remove these kind of comments.